Scripts/CI/README_Metadata_StyleCheck/action.yml

```diff
@@ -0,0 +1,20 @@
+# The name of the step just as a reference.
+name: "mdl and Python style checks"
+description: "This check will run several scripts to ensure the formatting."
+author: "tchen@esri.com"
```

Comment:
Should we update the email here at all?

Reply:
I'll run this by Ting and how we can handle across each SDK